docs/getting-started.md

```diff
@@ -221,6 +221,58 @@ lhci autorun
 
 </details>
 
+<details>
+<summary>Google Cloudbuild</summary>
+<br />
+
+**NOTE:** Learn more about the [security tradeoffs](./recipes/docker-client/README.md#--no-sandbox-container-tradeoffs) behind use of the `--no-sandbox` Chrome option before proceeding.
+
+**.lighthouserc.js**
+
+```js
+module.exports = {
+  ci: {
+    collect: {
+      settings: {chromeFlags: '--no-sandbox'},
+    },
+    upload: {
+      target: 'temporary-public-storage',
+    },
+  },
+};
+```
+
+**Notes**
+* `LHCI_BUILD_CONTEXT__CURRENT_BRANCH` doesn't pick up the right variables in cloudbuild so passing through `$BRANCH_NAME` will fix this.
+* `machineType` defines the machine you can pick. The bigger the better (see: https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md). Look through the [Cloudbuild machine pricing](https://cloud.google.com/cloud-build/pricing)
+
+**cloudbuild.yml**
+
+```yaml
+steps:
+  - id: 'install'
+    args: ['npm', 'ci']
+    name: node:14-alpine
+
+  - id: 'build'
+    waitFor: ['install']
+    name: node:14-alpine
+    args: ['npm', 'run', 'build']
+
+  - id: 'lighthouse'
+    waitFor: ['build']
+    name: cypress/browsers:node14.15.0-chrome86-ff82
+    entrypoint: '/bin/sh'
+    args: ['-c', 'npm install -g @lhci/cli@0.6.x && lhci autorun --failOnUploadFailure']
+    env:
+      - 'LHCI_BUILD_CONTEXT__CURRENT_BRANCH=$BRANCH_NAME'
+
+options:
+  machineType: 'N1_HIGHCPU_32'
```

Comment:
```suggestion
  machineType: 'N1_HIGHCPU_8'
```

WDYT about starting with 8 by default? the getting started guide isn't meant for folks to be relying on the stability of metric results from the get-go and maybe we add a note to the later stages about increasing the machine type (for whatever your CI is)? 

that way all the examples are fairly consistent too since I know the other machine types in other CI examples won't have 32 cores :)

Reply:
Yeah agreed. We didn't test at that due to our build having many other bits to it. I think good to have it at 8 and mention the ability to improve. 

Happy to take a look at the other CIs and try and document something here